<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1391</width>
    <height>752</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>271</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="title">
     <string>Session</string>
    </property>
    <widget class="QLineEdit" name="input_ip">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>166.111.80.237</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="input_port">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>113</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>8279</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ip">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="text">
      <string>host IP:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_port">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="text">
      <string>port:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_login">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Local</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>64</width>
      <height>15</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
    <property name="text">
     <string>Remote</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_local">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>681</width>
      <height>561</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QListWidget" name="list_remote">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>120</y>
      <width>681</width>
      <height>571</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier New</family>
     </font>
    </property>
   </widget>
   <widget class="QTreeView" name="tree_local">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>681</width>
      <height>571</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressbar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>698</y>
      <width>1381</width>
      <height>20</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh_remote">
    <property name="geometry">
     <rect>
      <x>1100</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_refresh_local">
    <property name="geometry">
     <rect>
      <x>990</x>
      <y>0</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Refresh</string>
    </property>
   </widget>
   <zorder>btn_refresh_remote</zorder>
   <zorder>groupBox</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>list_local</zorder>
   <zorder>list_remote</zorder>
   <zorder>tree_local</zorder>
   <zorder>progressbar</zorder>
   <zorder>btn_refresh_local</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
